src/controllers/things_controller.js

```diff
@@ -63,6 +63,28 @@ ThingsController.get('/', (request, response) => {
   }
 });
 
+ThingsController.patch('/', async (request, response) => {
```

Comment:
Why is thingId part of the body instead of part of the URL?

Reply:
Because the thing hasn't actually been added to the gateway yet. It's in the weird limbo state between "pairing" and "saving".